cmd/run_controller/main.go

```diff
@@ -132,16 +155,21 @@ func main() {
 	}
 	controller := runctl.NewController(factory, controllerOpts)
 
-	klog.V(3).Infof("Create Signal Handlers")
+	klog.V(3).InfoS("Create Signal Handlers")
 	stopCh := signals.SetupShutdownSignalHandler()
 	signals.SetupThreadDumpSignalHandler()
 
-	klog.V(2).Infof("Start Informer")
+	klog.V(2).InfoS("Start Informer")
 	factory.StewardInformerFactory().Start(stopCh)
 	factory.TektonInformerFactory().Start(stopCh)
 
-	klog.V(2).Infof("Run controller (threadiness=%d)", threadiness)
+	klog.V(2).InfoS("Run pipeline run controller", "threadiness", threadiness)
 	if err = controller.Run(threadiness, stopCh); err != nil {
-		klog.Fatalf("Error running controller: %s", err.Error())
+		klog.ErrorS(err, "Failed to run controller")
+		timeoutAndExit(errorExitCode)
 	}
 }
+
+func timeoutAndExit(exitCode int) {
```

Comment:
The name is hard to understand. Better: `flushLogsAndExit`.

Reply:
Done b7ba06e8160ab4760f214fda15639a6773ceca5c